devbox.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://raw.githubusercontent.com/jetify-com/devbox/main/.schema/devbox.schema.json",
+  "packages": [
+    "python@3.12",
+    "go@1.23",
+    "php@8.3",
+    "php83Packages.phpstan@1.11",
+    "python312Packages.mypy@1.10",
+    "golangci-lint@1.61",
+    "php83Packages.psalm@5.25",
+    "nodejs_18@18.20",
+    "nodePackages.ts-node@10.9.2",
+    "jre17_minimal@17",
+    "gradle@8.10"
+  ],
+  "shell": {
+    "init_hook": [
+      "echo 'Installing dependencies...' && make deps"
+    ],
+    "scripts": {
+      "go-example":  "make run-go-example",
+      "java-example": "make run-java-example",
+      "php-example": "make run-php-example",
+      "ts-example":  "make run-ts-example",
+      "lint":        "make lint",
+      "tests":       "make tests",
+      "gen-sdk-dev": "make gen-sdk-dev"
```

Comment:
Python is missing on purpose?

Reply:
For some reason we didn't run the python examples in the CI before. I didn't fix that in this PR, I'll do it in another one if we decide to merge this :)

Reply:
I tested the rest of them and they are working :)